app/views/user_mailer/shared/_in_person_warning_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<table class="usa-alert usa-alert--warning margin-bottom-4">
+  <tbody>
+  <tr>
+    <td style="width:16px;">
+      <%= image_tag('email/warning.png', width: 16, height: 14, alt: 'warning icon', style: 'margin-top: 5px;') %>
```

Comment:
[non-blocking] do you think it would be better to use an svg here or is the png ok? I just saw the styling and thought maybe it would be a bit easier to work with as an svg.

Reply:
We have to use pngs for email because svgs are not supported by most email clients.

see https://github.com/18F/identity-idp/pull/10839#discussion_r1648075095

Reply:
For the alt text, do you want to ask if we can get translations order for that? Else you can use 
`two_factor_authentication.important_alert_icon` with is important alert icon